frontend/src/bundles/common/components/badge/badge.tsx

```diff
@@ -12,20 +12,23 @@ type Properties = {
     description: string;
     secondText?: string;
     color?: ValueOf<typeof BadgeColors>;
-    small?: boolean | undefined;
+    isSmall?: boolean;
     icon?: ReactElement;
+    iconClass?: string;
+    isHRbadge?: boolean;
```

Comment:
we should rename it. 
Today it is HR, in the future it can be something else

Reply:
done